deployment/build.linux.amd64

```diff
@@ -9,7 +9,11 @@ set -o xtrace
 pushd ${SOURCE_DIR}
 
 # Get version
-version="$( grep "version:" ./build.yaml | sed -E 's/version: "([0-9\.]+.*)"/\1/' )"
+if [[ ${IS_UNSTABLE} == 'yes' ]]; then
+    version="${BUILD_ID}"
```

Comment:
why this doesn't have all the fancy things about getting a PR number?

Reply:
There's no changelog to store it to in the portable archives. That would be a cool idea, but it's outside of scope for this PR.